src/test_typing_extensions.py

```diff
@@ -5519,6 +5578,19 @@ def barfoo3(x: BA2): ...
             BA2
         )
 
+    @skipUnless(sys.version_info >= (3, 11), "TODO: evaluate nested forward refs in Python < 3.11")
```

Comment:
```suggestion
    @skipUnless(TYPING_3_11_0, "TODO: evaluate nested forward refs in Python < 3.11")
```

Use the constant for consistency?